payment/templates/payment/email/base.html

```diff
@@ -42,7 +42,7 @@
         <table border="0" width="100%" cellpadding="0" cellspacing="0">
           <tr>
             <td valign="middle" style="padding:10px 0; text-align:left;" width="150">
-              <a href="https://www.fun-mooc.fr"><img style="margin-left: 8px;" src="https://www.fun-mooc.fr/static/bulk_email/images/logo-funmooc.png" alt="alt text" width="80" height="80" border="0" align="left"></a>
+              <a href="https://www.fun-mooc.fr"><img style="margin-left: 8px;" src="${'https://{lms_base}/static/bulk_email/images/logo-funmooc.png'.format(lms_base=settings.LMS_BASE)}" alt="alt text" width="80" height="80" border="0" align="left"></a>
```

Comment:
It seems you also have to fix lines 84 & 85.

Reply:
Well spotted :astonished:
Thank you, it is now fixed :pray: